integration_tests/samples/openid_connect/flask_example.py

```diff
@@ -0,0 +1,118 @@
+import json
+import logging
+import os
+import jwt
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+client_id = os.environ["SLACK_CLIENT_ID"]
+client_secret = os.environ["SLACK_CLIENT_SECRET"]
+redirect_uri = os.environ["SLACK_REDIRECT_URI"]
+scopes = ["openid", "email", "profile"]
+
+from slack_sdk.web import WebClient
+from slack_sdk.oauth import OpenIDConnectAuthorizeUrlGenerator, RedirectUriPageRenderer
+from slack_sdk.oauth.state_store import FileOAuthStateStore
+
+state_store = FileOAuthStateStore(expiration_seconds=300)
+
+authorization_url_generator = OpenIDConnectAuthorizeUrlGenerator(
+    client_id=client_id,
+    scopes=scopes,
+    redirect_uri=redirect_uri,
+)
+redirect_page_renderer = RedirectUriPageRenderer(
+    install_path="/slack/install",
+    redirect_uri_path="/slack/oauth_redirect",
+)
+
+
+from flask import Flask, request, make_response
+
+app = Flask(__name__)
+app.debug = True
+
+
+@app.route("/slack/install", methods=["GET"])
+def oauth_start():
+    state = state_store.issue()
+    url = authorization_url_generator.generate(state=state)
+    return (
+        '<html><head><link rel="icon" href="data:,"></head><body>'
+        f'<a href="{url}">'
+        f'<img alt=""Add to Slack"" height="40" width="139" src="https://platform.slack-edge.com/img/add_to_slack.png" srcset="https://platform.slack-edge.com/img/add_to_slack.png 1x, https://platform.slack-edge.com/img/add_to_slack@2x.png 2x" /></a>'
+        "</body></html>"
+    )
+
+
+@app.route("/slack/oauth_redirect", methods=["GET"])
```

Comment:
Interesting that you are using the same routes we use for normal oauth

Reply:
Not strong opinion here 😸 If you prefer other paths for tutorials, we can update this example for consistency